File: Assets/javascripts/Spinning.js
#pragma strict

public var speed:float=1000;

function Start(){
}

function Update()
{
	if(Input.GetKey("w")){
		transform.Rotate(Vector3.right, speed * Time.deltaTime*100);
		speed+=Time.deltaTime;
	}
	if(Input.GetKey("s")){
		transform.Rotate(Vector3.left, speed * Time.deltaTime*100);
		speed+=Time.deltaTime;
	}
}

File: Assets/scripts/spinning.cs
using UnityEngine;

public class Spinning : MonoBehaviour
{
    public float maxSpeed = 2000f;
    

    void Update()
    {
        if (Input.GetKey(KeyCode.W))
        {
            transform.Rotate(Vector3.right, speed * Time.deltaTime * 100);
            speed = Mathf.Min(speed + Time.deltaTime, maxSpeed);
        }
        if (Input.GetKey(KeyCode.S))
        {
            transform.Rotate(Vector3.left, speed * Time.deltaTime * 100);
            speed = Mathf.Min(speed + Time.deltaTime, maxSpeed);
        }
    }
}